python/lsst/cp/pipe/defects.py

```diff
@@ -716,3 +861,70 @@ def run(self, inputDefects, camera):
         return pipeBase.Struct(
             mergedDefects=merged,
         )
+
+# Subclass the MergeDefects task to reduce the input dimensions
+# from ("instrument", "detector", "exposure") to
+# ("instrument", "detector").
+
+
+class MergeDefectsCombinedConnections(MergeDefectsConnections,
```

Comment:
This should not inherit from MergeDefectsConnections, it has different connections.  It should just inherit from the real base class.